game.go

```diff
@@ -1335,3 +1335,16 @@ func (p *Game) ShowVar(name string) {
 }
 
 // -----------------------------------------------------------------------------
+// Widget
+
+// GetWidget returns the widget instance with given name. It panics if not found.
+func (p *Game) GetWidget(name string) Widget {
+	items := p.items
+	for _, item := range items {
+		widget, ok := item.(Widget)
+		if ok && widget.GetName() == name {
+			return widget
+		}
+	}
+	return nil
```

Comment:
> It panics if not found.

We expect it to panic here, to keep consistent with other similar operations, e.g. `turnTo("not_exist")`